app/services/foreman/unattended_installation/mac_list_extractor.rb

```diff
@@ -0,0 +1,32 @@
+module Foreman
+  module UnattendedInstallation
+    class MacListExtractor
```

Comment:
Can be named AnacondaMacHeaderExctractor or maybe Detector not sure. This is not standardized header, it's Anaconda only.

Reply:
Since its already namespace'd to `UnattendedInstallation` isn't this ok as is ?

Reply:
I'd keep the current name as this also gets the mac address from the request parameter (e.g. for iPXE, `/unattended/iPXE?mac=...`)

Reply:
Well I wanted to give emphasis on the fact that this header is only sent by Anaconda, but not big deal. Users are actually using this outside of Anaconda context anyway.